src/test/java/uk/gov/hmcts/reform/ccd/client/CcdClientTest.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.hmcts.reform.ccd.client;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+import com.google.common.io.Resources;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.Classification;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.okJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@SpringBootTest(classes = {
+        CoreCaseDataClientAutoConfiguration.class,
+        CoreCaseDataConfiguration.class,
+        CoreCaseDataApi.class
+})
+@ExtendWith(SpringExtension.class)
+@EnableAutoConfiguration
+class CcdClientTest {
+
+    private static final LocalDateTime DATE_TIME = LocalDateTime.of(2018, 11, 01, 14, 37, 47, 584000000);
+    private static WireMockServer wireMockServer;
+
+    @Autowired
+    private CoreCaseDataApi ccdApi;
+
+    @BeforeAll
+    static void configureSystemUnderTest() {
+        wireMockServer = new WireMockServer(WireMockConfiguration.wireMockConfig().dynamicPort());
+        wireMockServer.start();
+        WireMock.configureFor(wireMockServer.port());
+        System.setProperty("core_case_data.api.url", "http://localhost:" + wireMockServer.port() + "/");
+    }
+
+    @AfterAll
+    static void afterAll() {
+        wireMockServer.shutdownServer();
+    }
+
+    @SneakyThrows
+    private static String loadFile(String filename) {
+        return Resources.toString(ClassLoader.getSystemClassLoader().getResource(filename), UTF_8);
+    }
+
+    @Test
+    @DisplayName("Should be able to call the new Api")
```

Comment:
can you remove new? not really that useful when someone looks at it later
```suggestion
    @DisplayName("Should be able to retrieve a case")
```

Reply:
sure ... was not thinking forward when I wrote it.